Sources/SnapshotTesting/SnapshotsTestTrait.swift

```diff
@@ -0,0 +1,52 @@
+#if canImport(Testing)
+  @_spi(Experimental) import Testing
+
+  @_spi(Experimental)
+  extension Trait where Self == _SnapshotsTestTrait {
+    /// Configure snapshot testing in a suite or test.
+    ///
+    /// - Parameters:
+    ///   - record: The record mode of the test.
+    ///   - diffTool: The diff tool to use in failure messages.
+    public static func snapshots(
+      record: SnapshotTestingConfiguration.Record = .missing,
+      diffTool: SnapshotTestingConfiguration.DiffTool = .default
```

Comment:
@mbrandonw Do you know if traits cascade and if this means these should be optional so that `.missing` won't get applied where a parent suite set to `.all`?